Scripts/gen_Primary.sh

```diff
@@ -148,18 +148,22 @@ elif [[ "${TYPE}" == "Piplus" ]]; then
   resampler=TargetPiStopResampler
 elif [[ "${TYPE}" == "Piminus" ]]; then
   resampler=TargetPiStopResampler
-  dataset=sim.${OWNER}.PiMinusFilter.${STOPS_CAMPAIGN}.art # since we prefilter these for a given time
+  dataset=sim.${OWNER}.${TYPE}StopsFilt.${STOPS_CAMPAIGN}.art # since we prefilter these for a given time
```

Comment:
This will not work with your current dataset.
You can drop this line altogether, if you plan to remake the dataset with a new name.